src/mumble/ConnectDialog.h

```diff
@@ -322,7 +347,7 @@ class ConnectDialog : public QDialog, public Ui::ConnectDialog {
 		~ConnectDialog();
 
 #ifdef USE_BONJOUR
-	protected:
+protected:
```

Comment:
```suggestion
	protected:
```
why did you un-indent this?

Reply:
Must have been auto indentation...